### DISCUSSION TITLE
ESLint not using .prettierrc in root directory when "Setup eslint?" option selected at install

### DISCUSSION THREAD

**Discussion by bsdev90** [adonisjs/core#3710] [Help] [answered]
I'm new to AdonisJS and after watching some tutorials I wanted to start my first project.
Having seen the benefits of ESLint/Prettier and having tested it, I answered "_Yes_" to the question "_Setup eslint?_" when installing AdonisJS.

I configured them (by installing `eslint-config-prettier` and `eslint-plugin-prettier` in addition to `eslint` and `prettier` and then adding `"plugin:prettier/recommended"` in the `.eslintrc.json` file) but it doesn't work as I want:
- I created a controller via the `make:controller` command but when I run the `eslint . --ext .ts` command, it doesn't take into account my Prettier configuration in the `.prettierrc` file located at the root of the project.
- On the other hand, if I copy this `.prettierrc` file in `app/Controllers/HTTP` directory, ESLint finally takes into account my Prettier configuration.

So I tried to recreate a project but this time by selecting "_No_" to the question "_Setup eslint?_", put back my ESLint and Prettier configuration, recreated my controller and... everything works, no need to take out the `.prettierrc` file from the root.

Is this a known problem? Did I miss something in the documentation?

Thanks in advance for your help and see you soon.

**Comment by thetutlage** [comment, marked as answer]
When you say "Yes" to ESLint/prettier prompt, then we configure them as per the conventions used by the core team. So maybe its possible that your manual config + our automated config are having some sort of conflict.

If you already have you conventions finalized, then you can simply say "No" and just use your config and plugins after creating the project

**Reply by bsdev90** [reply]
Oh ok thank you. I will surely use your conventions then, I just made some testing like `tabWidth` to 20 to verify that ESLint was reading my `.prettierrc` file (first time with these tools like I said before).
So:
- AdonisJS conventions: Yes to prompt
- Own conventions: No to prompt

Ready to develop my first AdonisJS app then ! From a PHP/Symfony guy, it seems the best NodeJS framework here :)
